Source/OxyPlot.Xamarin.Forms.Platform.UWP/Forms.cs

```diff
@@ -0,0 +1,28 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Forms.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Initializes OxyPlot renderers for use with Xamarin.Forms.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Xamarin.Forms.Platform.UWP
+{
+    using System;
+
+    /// <summary>
+    /// Initializes OxyPlot renderers for use with Xamarin.Forms.
+    /// </summary>
+    public static class Forms
```

Comment:
I thought it might be relevant for consistency with the wp8 project, but I removed it.